Add Mini Apps series (#3938)
GoogleChrome/web.dev: src/site/content/en/mini-apps/mini-app-conclusion/index.md

---
layout: post

> title: Conclusion

Reviewer: Also not going to yield strong SEO results. "Concluding thoughts about mini apps from a web developer" or something like that would probably perform better

Author: Went with your suggestion!

> title: Conclusion

Author: ```suggestion
title: Concluding thoughts about mini apps from a web developer
```

My initial experiments with
[building a web application the mini app way](mini-app-example-project/) were
successful. Future work will show to what extent this model is performant and flexible enough to
cater for the many shapes that web apps can take. My current _ad-hoc_ approach can be formalized by
packaging up the relevant pieces of code in a dedicated library, `mini-app.js` if you will. What is
interesting is that this kind of programming goes back all the way to `frameset`. Just that today it
is about applications and not documents.

I see great potential for improvement with the entire web development experience by taking
inspiration from the various mini apps DevTools. From the easy
[(remote) on-device testing feature](/mini-app-devtools/#simulator-and-real-device-testing-and-debugging)
to the packaging and [building](/mini-app-project-structure-lifecycle-and-bundling/#the-build-process)
experience; the integration of the [IDE](/mini-app-devtools/#mini-app-ides) with the DevTools
environment offers a lot of starting points for making developers' lives easier.

## Closing thoughts

> From a features point of view, the web is catching up with mini apps. The ever-growing

Reviewer: Please rephrase this first sentence. I'm hearing a lot of talk about wanting to get away from this narrative of the web catching up to iOS/Android or any other platform. Ping me on Chat if you want to discuss.

Author: Done. Made this focused on browser releases, not other platforms.

> From a features point of view, the web is catching up with mini apps. The ever-growing

Author: ```suggestion
From a features point of view, the web is becoming more and more powerful. The ever-growing
```

[list of capabilities](/fugu-status/) makes use cases possible on the web that were unthinkable a
mere year ago. At the same time, the need for
[mini apps standardization](/mini-app-standardization/) shows that developers are not willing or
able to build the same mini app for each super app. On the horizon maybe there is a desire for an
abstraction layer on the browser level that allows for mini apps to run on the web, while noting that
the web is not immune from fragmentation, especially when looking at different browser vendors and
what they choose to implement and what not. Concluding, I am looking forward to seeing where all
this is headed. Thinking outside of the box and taking input and inspiration from outside of one's
own bubble can definitely help when building a better future on the web.

{% Banner 'neutral' %}
  👉 Congratulations, you have reached the end of the [mini apps collection](/mini-apps/).

> {% endBanner %}

Reviewer: Bravo 🎆

Author: 🎉 Yay!
